services/src/main/java/io/druid/cli/convert/ConvertProperties.java

```diff
@@ -68,7 +68,16 @@ private static String parseJdbcUrl(String jdbcUrl){
       new Rename("druid.db.connector.connectURI", "druid.metadata.storage.connector.connectURI"),
       new Rename("druid.db.connector.user", "druid.metadata.storage.connector.user"),
       new Rename("druid.db.connector.password", "druid.metadata.storage.connector.password"),
+      new Rename("druid.db.connector.validationQuery", "druid.metadata.storage.connector.validationQuery"),
+      new Rename("druid.db.connector.useValidationQuery", "druid.metadata.storage.connector.useValidationQuery"),
```

Comment:
Validation query configs should be removed. It's hard coded in the storage module and always used by default in 0.7